relentless/potential/potential.py

```diff
@@ -529,7 +531,54 @@ def __next__(self):
         return next(self.coeff)
 
 class Tabulator:
-    def __init__(self, nbins, rmin, rmax, fmax=None, fcut=None, edges=True):
+    """Combines and tabulates multiple potentials together.
+
+    Evaluates accumulated energy and force values for multiple potential functions at different r values,
+    allows regularization of the force and shifting of the energy.
+
+    Parameters
+    ----------
+    nbins : int
+        The number of bins in which to place the values for the potential.
```

Comment:
I realize now why you used "bins" throughout... this was a bad name choice on my part! I probably borrowed it from an RDF code or numpy.histogram. I think we should actually give the user more flexibility here. Rather than specifying `nbins`, `rmin`, `rmax`, and `edges`, let's just let the user give an array like `r`. Internally, copy it into a numpy array, exposed using getters and setters:
```py
@property
def r(self):
    return self._r
@r.setter
def r(self, points):
    points = np.array(points)
    # error check shape and order here
    self._r = points
```
The error check should be that `r` is a 1d array and that its entries are continuously increasing. In the constructor, you just need:
```py
self.r = r
```
Then when constructing the `Tabulator`, the user can use `linspace` or whatever method they like (for example, if they want irregular spaced points).